src/auth/ui/vue/root.vue

```diff
@@ -30,6 +30,36 @@
                 </div>
             </div>
         </div>
+
+        <div
+            v-if="successfulAuthConnection"
+            class="border-common"
+            style="
+                width: fit-content;
+                white-space: nowrap;
+                display: flex;
+                flex-direction: row;
+                background-color: #28632b;
+                color: #ffffff;
+                padding: 10px;
+            "
+        >
+            <div class="icon icon-lg icon-vscode-check"></div>
+            &nbsp; &nbsp;
+            <div style="display: flex; flex-direction: row">
+                You're connected to {{ authFormDisplayName }}! Switch between connections in the&nbsp;<a
+                    v-on:click="showConnectionQuickPick()"
+                    style="cursor: pointer"
+                    >Toolkit panel</a
+                >&nbsp;or add additional connections below.
```

Comment:
"additional" is a verbose way to say "more"

```suggestion
                >&nbsp;or add more connections below.
```